bin/appliance_console

```diff
@@ -435,7 +435,6 @@ Static Network Configuration
             raise MiqSignalError
           elsif message_server.ask_questions && message_server.configure
             say("\nMessage Server configured successfully.\n")
-            press_any_key
           else
             say("\nMessage Server configuration failed!\n")
             press_any_key
```

Comment:
Will it also prompt twice on failure?  I assume it's the `ask_questions` method that is prompting the first time?